buildbot/configure.py

```diff
@@ -371,7 +394,7 @@ def main():
     )
     parser.add_argument(
         "--cmake-opt",
-        action="append",
+        action=_DeprecatedCmakeOpt,
         help="Additional CMake option not configured via script parameters",
```

Comment:
Should we also update the message like:
```suggestion
        help="DEPRECATED: use -D instead. Additional CMake option not configured via script parameters",
```
?

Reply:
Actually, this is not just limited to `-D`. I'll change it to just add "DEPRECATED"